pyriksdagen/refine.py

```diff
@@ -354,6 +354,25 @@ def detect_date(root, metadata):
     protocol_years = {protocol_year, metadata.get("secondary_year", protocol_year)}
     yearless = set()
 
+    def _is_sjukbetyg(elem):
+        """
+        check if elem with date is part of a (lakar|sjul)(be|in)tyg
+        """
+        found_title = False
+        is_sjukbetyg = False
+        pat = re.compile(r'(läkar|sjuk)(in|be)tyg')
+        while found_title == False:
```

Comment:
What do you mean by "work"? if there's no title above the found date, it will iterate in reverse to the beginning of the protocol and then break.
![image](https://github.com/user-attachments/assets/7960ab2f-6b9e-4030-a80c-de8860deedb4)

Reply:
Does break actually break the loop or just the if statement? I think I have had issues with that